CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Reactist follows [semantic versioning](https://semver.org/) and doesn't introduce breaking changes (API-wise) in minor or patch releases. However, the appearance of a component might change in a minor or patch release so keep an eye on redesigns and make sure your app still looks and feels like you expect it.
 
+# v21.2.1
+
+-   [Chore] Fix links to Reactist components in storybook stories.
```

Comment:
😅 Don't think we typically use "[Chore]" in this changelog, apologies, will update.